iphone/cli/commands/_build.js

```diff
@@ -3643,8 +3654,6 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 
 						if (legacySwift) {
 							extBuildSettings.EMBEDDED_CONTENT_CONTAINS_SWIFT = 'YES';
-						} else {
-							extBuildSettings.ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = 'YES';
 						}
```

Comment:
This was an old left over from the Swift 2 times. Nowadays, swift support us enabled by default and the build manages the `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` flag on a module level.